receiver/collectdreceiver/collectd.go

```diff
@@ -59,6 +59,13 @@ func (r *collectDRecord) protoTime() *timestamppb.Timestamp {
 	return timestamppb.New(ts)
 }
 
+func (r *collectDRecord) startTimestamp(mdType metricspb.MetricDescriptor_Type) *timestamppb.Timestamp {
+	if mdType == metricspb.MetricDescriptor_CUMULATIVE_DISTRIBUTION || mdType == metricspb.MetricDescriptor_CUMULATIVE_DOUBLE || mdType == metricspb.MetricDescriptor_CUMULATIVE_INT64 {
+		return timestamppb.New(time.Unix(0, int64(float64(time.Second)**r.Time)-int64(float64(time.Second)**r.Interval)))
```

Comment:
Something I don't like about "**" can you please refactor the code to avoid that?

Reply:
Thanks @bogdandrutu! I committed a change that should improve the readability of this code. Let me know if it looks better.